R4ML/R/ml.model.svm.R

```diff
@@ -150,24 +151,24 @@ setMethod("initialize", "r4ml.svm",
 #' @examples \dontrun{
 #' 
 #' # Load the Iris dataset 
-#' iris_hf <- as.r4ml.frame(iris)
+#' iris_r4mlDF <- as.r4ml.frame(iris)
```

Comment:
I agree that it hf and hm are very confusing since all references to hydra have been removed.  I can change it to r4ml_df if you like that better.  The other change(rm -> r4mlMat) is okay then?